Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
Instead of doing this, let's do it like this:

```ruby

if build.with? "static"
  args << "--disable-shared"
else
  args << "--enable-shared"
end
```

That way we don't need to duplicate the whole block.